go.sum

```diff
@@ -32,7 +32,6 @@ github.com/davecgh/go-spew v1.1.1 h1:vj9j/u1bqnvCEfJOwUhtlOARqs3+rkHYY13jYWTU97c
 github.com/davecgh/go-spew v1.1.1/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
 github.com/diegoholiveira/jsonlogic/v3 v3.2.2 h1:qV3mShglvFIwnr37ZB6kL81Gk5vHrGDevQGy2j1AMQA=
 github.com/diegoholiveira/jsonlogic/v3 v3.2.2/go.mod h1:9oE8z9G+0OMxOoLHF3fhek3KuqD5CBqM0B6XFL08MSg=
-github.com/eclipse/paho.mqtt.golang v1.3.5/go.mod h1:eTzb4gxwwyWpqBUHGQZ4ABAV7+Jgm1PklsYT/eo8Hcc=
```

Comment:
I notice no go.mod change these maybe made it back in on a merge or something?

Reply:
This is caused by the go-mod-messaging dependency. Not sure why it didn't update to go.mod when I ran `tidy`. Ran it again and no change...